src/MissingRequiredArgumentException.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+declare(strict_types=1);
```

Comment:
Do we need to handle InvalidParameterException  and MissingRequiredArgumentException separately?

Maybe it's enough to have \InvalidArgumentException only?

Reply:
Decided to leave as is for the possibility of introducing friendly exception.